### ISSUE TITLE
Ubuntu Server: CSS styling of glossary terms

### ISSUE THREAD

**Issue by s-makin** [canonical/open-documentation-academy#189]
## Task

The [Ubuntu Server glossary](https://documentation.ubuntu.com/server/reference/glossary/) is shaping up beautifully! Thanks to the hard work of the CODA community, who [have implemented it](https://github.com/canonical/ubuntu-server-documentation/pull/124), have been filling in definitions (too many PRs for me to link!), working on [a format we can apply to definitions](https://github.com/canonical/open-documentation-academy/pull/184), and we now have the [hover-over functionality](https://github.com/canonical/ubuntu-server-documentation/pull/193) - this is awesome work, and it's really elevating the documentation.

The last step before we can document "how to set up a glossary" is to address the styling of the glossary terms and the hover-over box. 

This issue should be relatively straightforward, but will require some knowledge of how CSS works. 

## Details

This page should provide a [helpful test example](https://documentation.ubuntu.com/server/explanation/virtualisation/vm-tools-in-the-ubuntu-space/). Right at the top of the page, under KVM, is an external link just above a glossary term "AMD" (good for easy side-by-side comparison).

Please feel free to experiment with the styling, and present whatever you think looks best! We currently have no defined standard in our style guide, so everything is on the table!
* Hover box:
  * (required) increase the contrast - this pale grey is not good from an accessibility point of view.
  * (optional) whatever other changes you think look good.
* Glossary term links: 
  * (required) The style should be subtly different from the external link styling so that users can quickly learn which ones will give them a hover-over definition.
  * (optional) colour (before/after clicking), font weight, underlining, etc can all be played with!
* All our CSS files are [held in this directory](https://github.com/canonical/ubuntu-server-documentation/tree/main/.sphinx/_static).
   
It's a good idea to create a new/separate CSS file for this update in that directory. You may also need to add the name of the file you create to the [custom_conf.py file](https://github.com/canonical/ubuntu-server-documentation/blob/main/custom_conf.py) so Sphinx can find it.

You can claim this issue here, but make sure to submit your PR against the [Ubuntu Server](https://github.com/canonical/ubuntu-server-documentation) repository.

### Context

The [Ubuntu Server docs](https://github.com/canonical/ubuntu-server-documentation) were recently migrated to GitHub/Read the Docs from their previous home on Discourse. We'd be grateful for any help on several issues that will help us to finalise this and provide a better experience for our contributors and readers.

### Useful links

- Check out the Ubuntu Server [contributing guide](https://documentation.ubuntu.com/server/contributing/).
- If you are new to the Canonical Open Documentation Academy, you may also want to review the [Getting Started content](https://discourse.ubuntu.com/t/getting-started/42769).
- If there are specific things you would like to learn, or want more experience on, I'm happy to bring forward other issues we have that fit with your goals.

Thanks in advance for your help!

**Comment by davidekete** [comment]
Hi @s-makin, I'd like to work on this.

**Comment by s-makin** [comment]
> Hi [@s-makin](https://github.com/s-makin), I'd like to work on this.

Thanks for volunteering, I've assigned it to you now :)

**Comment by davidekete** [comment]
Hi @s-makin, the maintainers of the library that generates the tooltips plan to archive it soon because Readthedocs made it a feature. Would you consider changing to Readthedocs before we proceed with this issue?

**Comment by s-makin** [comment]
> Hi [@s-makin](https://github.com/s-makin), the maintainers of the library that generates the tooltips plan to archive it soon because Readthedocs made it a feature. Would you consider changing to Readthedocs before we proceed with this issue?

Hi @davidekete, at the moment there doesn't seem to be a straightforward method for customising the built-in RTD previews, but it looks like it's on their roadmap for this month. I think we should probably put a pin this for now - I'll close this issue, and once we know what customisation options are available we can reassess the landscape and reopen it (probably next month). Thanks for bringing it my attention! I'll give you "first refusal" when we reopen the issue.
